pubsublite-spark-sql-streaming/src/test/java/com/google/cloud/pubsublite/spark/PslSourceOffsetTest.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.pubsublite.spark;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.fail;
+
+import com.google.cloud.pubsublite.*;
+import com.google.common.collect.ImmutableMap;
+import org.junit.Test;
+
+public class PslSourceOffsetTest {
+
+  private static final SubscriptionPath TEST_SUB;
+
+  static {
+    TEST_SUB =
```

Comment:
not needed. just do "private static final SubscriptionPath TEST_SUB = ...

Reply:
I used to have it because I was using an old pubsub lite library that throws checked exception when I do ProjectId.of. Removed.